ajo/personalization-hybrid/src/server.js

```diff
@@ -65,6 +69,18 @@ function prepareTemplateVariables(
   defaultTemplateVariables = {}
 ) {
   const { headers = {}, body = { handle: [] } } = response;
+  const res =   response.body.handle.filter(
+    (item) => item.type === TYPE_PERSONALIZATION
+  )
+  const sanitizedPayload = res[0].payload.map((item) => ({
+    ...item,
+    items: item.items.map((subItem) => ({
+      ...subItem,
+      data: subItem.data ? sanitizeData(subItem.data) : null,
```

Comment:
is this really necessary?  I don't think it's reasonable to expect our customers to go to this trouble.

Can you explain the underlying problem in detail and how dompurify is solving it?

Reply:
When comparing the sanitized content with the original, I've noticed differences in the way certain characters are escaped. It's expected that DOMPurify, being a sanitizer, should handle escaping issues.